pkg/vgmanager/vgmanager_controller.go

```diff
@@ -244,6 +250,29 @@ func (r *VGReconciler) reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 	return ctrl.Result{RequeueAfter: requeueAfter}, nil
 }
 
+func (r *VGReconciler) addThinPoolToVG(vgName string, config lvmv1alpha1.ThinPoolConfig) error {
+	lvList, err := ListLogicalVolumes(r.executor, vgName)
+	if err != nil {
+		return fmt.Errorf("failed to list logical volumes in the volume group %q. %v", vgName, err)
+	}
+
+	for _, lv := range lvList {
+		if lv == config.Name {
+			r.Log.Info("logical volume is already created", "Volume Group", vgName, "Logical Volume", config.Name)
```

Comment:
```suggestion
			r.Log.Info("lvm thinpool already exists", "vgname", vgName, "thinpool", config.Name)
```

Reply:
An additional check to determine whether the existing lv is a thinpool or not would provide more info in the log message.